client/my-sites/marketing/business-tools/index.tsx

```diff
@@ -0,0 +1,216 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import MarketingBusinessToolsFeature from './feature';
+import MarketingBusinessToolsHeader from './header';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { recordTracksEvent as recordTracksEventAction } from 'calypso/state/analytics/actions';
+
+/**
+ * Images
+ */
+import quickbooksLogo from 'calypso/assets/images/illustrations/quickbooks-logo.svg';
+import evernoteLogo from 'calypso/assets/images/illustrations/evernote-logo.svg';
+import mondayLogo from 'calypso/assets/images/illustrations/monday-logo.svg';
+import benchLogo from 'calypso/assets/images/illustrations/bench-logo.svg';
+import todoistLogo from 'calypso/assets/images/illustrations/todoist-logo.svg';
+import streakLogo from 'calypso/assets/images/illustrations/streak-logo.png';
+import billcomLogo from 'calypso/assets/images/illustrations/billcom-logo.svg';
+
+/**
+ * Types
+ */
+import * as T from 'calypso/types';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	recordTracksEvent: typeof recordTracksEventAction;
+	selectedSiteSlug: T.SiteSlug | null;
+}
+
+export const MarketingBusinessTools: FunctionComponent< Props > = ( { recordTracksEvent } ) => {
+	const translate = useTranslate();
+
+	const handlePartnerClick = () => {
```

Comment:
If this list of handlers grows over time, it might be something to consider adding a more general click handler where we can pass in the event name as an argument. It looks like all of these are doing the exact same thing, the only difference being the event name.

Not a blocker at all for this PR at all, just a thought on DRYing things up a little bit.